src/main/java/net/minecraftforge/fmp/capabilities/CapabilityMicroblockContainerTile.java

```diff
@@ -0,0 +1,42 @@
+package net.minecraftforge.fmp.capabilities;
+
+import java.util.concurrent.Callable;
+
+import net.minecraft.nbt.NBTBase;
+import net.minecraft.util.EnumFacing;
+import net.minecraftforge.common.capabilities.Capability;
+import net.minecraftforge.common.capabilities.CapabilityInject;
+import net.minecraftforge.common.capabilities.CapabilityManager;
+import net.minecraftforge.fmp.microblock.IMicroblockContainerTile;
+
+public class CapabilityMicroblockContainerTile
```

Comment:
All of these classes should probably be a single class.... Just register all the caps at once... No need for all the seperated crap..